ports/stm32/boards/NUCLEO_WB55/rfcore_firmware.py

```diff
@@ -189,23 +194,30 @@ class _Flash:
     _FLASH_KEY1 = 0x45670123
     _FLASH_KEY2 = 0xCDEF89AB
 
+    _FLASH_CR_STRT_MASK = 1 << 16
+    _FLASH_CR_LOCK_MASK = 1 << 31
+
     def wait_not_busy(self):
-        while machine.mem32[stm.FLASH + stm.FLASH_SR] & 1 << 16:
+        while machine.mem32[stm.FLASH + stm.FLASH_SR] & _Flash._FLASH_CR_STRT_MASK:
```

Comment:
This should be `_FLASH_SR_BSY_MASK` (with value 1<<16)

Reply:
Done